external/pico_extras_import.cmake

```diff
@@ -33,7 +33,7 @@ if (NOT PICO_EXTRAS_PATH)
         if (NOT PICO_EXTRAS)
             message("Downloading PICO EXTRAS")
             FetchContent_Populate(PICO_EXTRAS)
-            set(PICO_EXTRAS_PATH ${PICO_EXTRAS_SOURCE_DIR})
+            set(PICO_EXTRAS_PATH ${pico_extras_SOURCE_DIR})
```

Comment:
Yeah this seems to match up with https://github.com/raspberrypi/pico-sdk/blob/master/external/pico_sdk_import.cmake#L40